### PR TITLE
bump ruby version from 3.3.0-rc1 to 3.3.0

### PR DESCRIPTION
**Pull request by tannalynn**
Use the official release of ruby version 3.3.0

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
       - name: Configure git
         run: 'git config --global init.defaultBranch main'
       - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # tag v3.5.0
-      - uses: ruby/setup-ruby@af848b40be8bb463a751551a1180d74782ba8a72 # tag v1.162.0
+      - uses: ruby/setup-ruby@360dc864d5da99d54fcb8e9148c14a84b90d3e88 # tag v1.165.1
         with:
           ruby-version: '3.2'
```
Comment (kaylareopelle) [inline .github/workflows/ci.yml:15]:
```suggestion
          ruby-version: '3.3'
```

**Review by kaylareopelle — COMMENTED** [review]
[This line](https://github.com/newrelic/newrelic-ruby-agent/pull/2381/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR35) references 3.2.2. Should it reference 3.3.0?

**Comment by tannalynn** [comment]
> [This line](https://github.com/newrelic/newrelic-ruby-agent/pull/2381/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR35) references 3.2.2. Should it reference 3.3.0?

oh yeah for sure, thank you i missed that one
should be updated now!

**Review by kaylareopelle — APPROVED** [review]
🦄

**Pull request merged**